monitoring/loadtest/Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.7
+# Not -alpine because: https://stackoverflow.com/a/58028091/651139
+# The context for this image should be monitoring
```

Comment:
This comment is wrong in a couple other Dockerfiles which is probably why it's here -- the context should be the root folder of the repo (per README and where the folders mentioned below are located).